### ISSUE TITLE
Is the config for the Panel available?

### ISSUE THREAD

**Issue by auraltension** [maK-/scantastic-tool#7]
I've just discovered this tool. Nice!  Good excuse for me to learn more ELK-foo.  I'm wondering if the Panel config is available? Or am I missing something very simple here....?

I've also created a Dockerfile to set this all up which I will hopefully publish tonight.

**Comment by maK-** [comment]
I have a few ansible scripts to set up my environment on a VPS but I won't be open sourcing the panel plugins just yet as they are a complete mess (this allowed running scans too from the kibana panel). 

The elasticsearch/kibana config I normally go with is generally default and runs fully on the localhost. Just make sure you specify the elasticsearch port you decide to run it on with the -esp flag.

I can add in notes for your dockerfile setup too if it's decent ;)

I basically want to avoid having to maintain the project as I'm quite busy as it is, why I left environment setup out and also why the code is such a mess ha

**Comment by auraltension** [comment]
Fair enough re the panel plugins. I noticed you had the ability to run scans from it which was my main motivation as I was thinking of doing something similar.

My basic Dockerfile is at https://gist.github.com/auraltension/c3e9df00fd32cb664264a3c6fa9d4fe9

Getting up and running is presently a case of:

```
docker build -t "scantastic:dockerfile" .
docker run scantastic:dockerfile
``

Kibana will be available on localhost:5601

To run scans I open a bash shell inside the container:

````
sudo docker exec -it scantastic:dockerfile bash 
cd scantastic-tool
./scantastic.py -s -H <ip> -p <ports> -x <file>.xml -esp 9200 -i logstash-<value>
```

I'll get it nicer and sort out some visualisations and ability to run scans from the browser and then put it into a repo.

I have a bunch of projects that are the same re maintenance so fully understand ;)

**Comment by maK-** [comment]
This is a great start! I've rarely used docker if I'm honest but it does seem the handiest/best way to go.
